### PR TITLE
Remove the `-ing` form from all BBE titles

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Remove the `-ing` form from all BBE titles.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by gimantha — APPROVED** [review]

File: examples/index.json [hunk 1]
```diff
@@ -295,7 +295,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Controlling openness",
+                "name": "Control openness",
                 "url": "controlling-openness",
```
Comment (MaryamZi) [inline examples/index.json:299]: Shouldn't these also change?

File: examples/index.json [hunk 2]
```diff
@@ -536,7 +536,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Ordering",
+                "name": "Order",
```
Comment (MaryamZi) [inline examples/index.json:539]: I don't think this makes sense. "Ordering" here is used as a noun.

**Review by MaryamZi — COMMENTED** [review]
@praneesha, why do we need this change? In some places it feels natural to have -ing.

**Comment by praneesha** [comment]
> @praneesha, why do we need this change? In some places it feels natural to have -ing.

@MaryamZi - We have removed the `ing` form in all the titles in all components of Ballerina documentation and this is to be consistent with that effort. This is done as we are complying with the Google style guide for Ballerina docs: https://developers.google.com/style/headings?hl=en

File: examples/index.json (hunk 1 above)
Comment (praneesha) [reply examples/index.json:299]: Yes. `Controlling openness` has been changed to `Control openness`. Do you mean we should change the URL as well?

File: examples/index.json (hunk 2 above)
Comment (praneesha) [reply examples/index.json:539]: Reverted to `Ordering`.

File: examples/index.json (hunk 1 above)
Comment (MaryamZi) [reply examples/index.json:299]: Yes.

Comment (praneesha) [reply examples/index.json]: Updated the URLs.

File: examples/index.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
         "samples": [
             {
                 "name": "Hello world main",
-                "url": "hello-world",
+                "url": "hello-world-main",
```
Comment (MaryamZi) [inline examples/index.json:9]: This isn't necessary, right?

File: examples/index.json [hunk 4]
```diff
@@ -211,7 +211,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Errors",
+                "name": "Error reporting",
```
Comment (MaryamZi) [inline examples/index.json:214]: Why are we changing these? I wouldn't change the titles to these.

File: examples/index.json [hunk 5]
```diff
@@ -550,15 +550,15 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Querying tables",
-                "url": "querying-tables",
+                "name": "tables querying",
```
Comment (MaryamZi) [inline examples/index.json:553]: Why are we changing this?

Comment (MaryamZi) [reply examples/index.json:553]: This is a title btw.

File: examples/index.json [hunk 6]
```diff
@@ -989,7 +989,7 @@
         "category": "Language concepts",
         "samples": [
             {
-                "name": "Assertions",
+                "name": "assertions",
```
Comment (MaryamZi) [inline examples/index.json:992]: Why are we starting with a lower case letter?

File: examples/index.json (hunk 3 above)
Comment (praneesha) [reply examples/index.json:9]: Reverted the URL to `hello-world`.

File: examples/index.json (hunk 4 above)
Comment (praneesha) [reply examples/index.json:214]: Changed to be consistent with the URL. Reverted the name back to `Errors`.

File: examples/index.json (hunk 5 above)
Comment (praneesha) [reply examples/index.json:553]: Reverted the name tback o `Querying tables` and the URL to `querying-tables`.

File: examples/index.json (hunk 6 above)
Comment (praneesha) [reply examples/index.json:992]: Sorry, that was a mistake. Fixed.

File: examples/index.json [hunk 7]
```diff
@@ -444,22 +444,22 @@
...
+                "url": "convert-from-user-defined-type-to-json",
                 "verifyBuild": true,
                 "verifyOutput": true,
                 "isLearnByExample": true
             },
             {
-                "name": "Converting from JSON to user-defined type",
-                "url": "converting-from-json-to-user-defined-type",
+                "name": "Convert from JSON to user-defined type",
+                "url": "convert-from-json-to-user-defined-type",
```
Comment (MaryamZi) [inline examples/index.json:455]: These will result in quite a few broken links, right? I'm not sure if that's acceptable given the number of BBEs we're changing.

File: examples/index.json [hunk 8]
```diff
@@ -700,15 +700,15 @@
         "category": "Language concepts",
         "samples": [
             {
-                "name": "Consuming services: client objects",
-                "url": "consuming-services",
+                "name": "Consume services: client objects",
+                "url": "consuming-services-client-objects",
```
Comment (MaryamZi) [inline examples/index.json:704]: Not required?

File: examples/index.json (hunk 7 above)
Comment (praneesha) [reply examples/index.json]: Reverted the URL changes.

File: examples/index.json (hunk 8 above)
Comment (praneesha) [reply examples/index.json]: Fixed.

File: examples/index.json [hunk 9]
```diff
@@ -578,7 +578,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Querying with streams",
+                "name": "Query with streams",
```
Comment (MaryamZi) [inline examples/index.json:581]: Here again I think "Querying" is more suitable.

**Review by MaryamZi — COMMENTED** [review]
Lang changes LGTM.

File: examples/index.json (hunk 9 above)
Comment (praneesha) [reply examples/index.json]: Reverted.

**Review by shafreenAnfar — APPROVED** [review]
I don't see any technical issues here.

File: examples/index.json [hunk 10]
```diff
@@ -975,7 +975,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Inferring isolated",
+                "name": "Infer isolated",
```
Comment (MaryamZi) [inline examples/index.json:978]: For this and L950 also, I think what we had previously is better.

Comment (praneesha) [reply examples/index.json]: Reverted both.

**Review by MaryamZi — APPROVED** [review]
Lang BBE changes LGTM.

**Pull request merged**